<launch>

<!-- MoveBaseSeq node launch and Points Sequence ROS Parameters Loading -->


    <!--Run move_base_seqs node-->
        <node pkg="autonomous_nav" type="send_goals.py" name="send_goals" output="screen">

        <!-- Load Sequence of Points (Cartesian Coordinates wrt "map" RF, in meters) -->
            <rosparam param="p_seq">[1,3, -6,0, 6,-1]</rosparam>
        <!-- Load Sequence of Desired Yaw Angles (no rotations around x and y axes) in degrees-->
            <rosparam param="yea_seq">[90,90,180]</rosparam>

        </node>

</launch>
